gravitl/netmaker-ui
# fix(GRA-1201): remove "local network"

TESTING STEPS:
1. Go to create/read/edit network and node, check that "is local network" or "local range" does not exist
